provision-osx/tasks/configure_base_system.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+-
+  stat:
+    path: ~/.bash_profile
+  register: bash_file
+
+-
+  name: Create user bash file
+  file:
+    path: ~/.bash_profile
+    state: touch
+  when: bash_file.stat.exists == False
+
+-
+  name: Configure bash profile language encoding
+  lineinfile:
+    dest: ~/.bash_profile
+    line: "export LANG={{ buildfarm_lang_env_var }}"
+
+-
+  name: Configure profile file
+  lineinfile:
+    dest: ~/.profile
+    line: "source ~/.bash_profile"
```

Comment:
I created a fresh new user in a mac machine and those bash files were not there (bashrc, bash_profile and profile), which is why I created this task - but yes, I think we can remove it for now and test it later with a fresh new user to check if those filer were created (or not).